src/main/java/seedu/address/logic/commands/FacebookConnectCommand.java

```diff
@@ -68,10 +70,15 @@ public static Facebook getFacebookInstance() {
     }
 
     /**
-     * Sets the authenticated Facebook instance
+     * Returns name of the authenticated user
+     */
+    public static String getAuthenticatedUsername() { return authenticatedUsername; }
```

Comment:
I think the `return authenticatedUsername` should be in the next line? Not 100% sure though